JSTOR.js

```diff
@@ -193,7 +221,7 @@ function processRIS(text, jid) {
 		}
 		
 		item.url = item.url.replace('http:','https:'); // RIS still lists http addresses while JSTOR's stable URLs use https
-		
+		if(!item.url.startsWith("http")) item.url = "https://" + item.url;
```

Comment:
To be safe, let's use `if (item.url && !item.url.startsWith("http"))`, in case the URL is ever empty.

(Also missing the space after `if` for the linter.)

Reply:
Ok also I just commented the last function because it threw an error (not used) should it just be deleted? For the copyright I used the names in the metadata and put 2019 as copyright year

Reply:
Yeah, it looks like that function can be removed after fb9e9e3d0ca4a6155b16d0f71940915d9e096004.

Reply:
Do you remember every commit or did you have to search? (if so how do you go about to find a commit on this problem as quickly?)

Reply:
`git blame` or the equivalent "Blame" button in the github gui is very helpful for this sort of thing.

Reply:
thanks this looks like a time saver

Reply:
In a case like this, where you want to find something that was deleted, `git log -S finalizeItem JSTOR.js` is a good option.